core/src/main/java/hudson/model/FileParameterDefinition.java

```diff
@@ -45,9 +45,18 @@
  * @author Kohsuke Kawaguchi
  */
 public class FileParameterDefinition extends ParameterDefinition {
+
+    /**
+     * @since TODO
+     */
     @DataBoundConstructor
+    public FileParameterDefinition(String name) {
+        super(name);
+    }
+
     public FileParameterDefinition(String name, String description) {
```

Comment:
Should we mark the other constructors as deprecated? (Repeat this for other parameter classes.)
```suggestion
    @Deprecated
    public FileParameterDefinition(String name, String description) {
```

Reply:
No strong opinion, but I did not see any particular reason to deprecate these constructors if they are working fine for functional tests and the like. I deprecated the _super_ constructor in `ParameterDefinition` as a reminder that subtypes (in plugin `src/main/java/`) ought to offer the simplified constructors.